embeddings/utils/utils.py

```diff
@@ -66,7 +70,10 @@ def _get_new_dir_name(embedding_or_dataset: T_path) -> str:
 
     embedding_name = _get_new_dir_name(embedding_name)
     dataset_name = _get_new_dir_name(dataset_name)
-    return Path(root, embedding_name, dataset_name)
+    path = Path(root, embedding_name, dataset_name)
+    if timestamp_subdir:
+        path = path.joinpath(f"{datetime.utcnow().strftime('%Y%m%d%H%M%S')}UTC")
```

Comment:
```suggestion
        path = path.joinpath(f"{datetime.utcnow().strftime('%Y%m%d_%H%M%S')}_UTC")
```
maybe add some formatting of the timestamp to facilitate reading and comparison of different runs

Reply:
done